src/coreclr/jit/instrsxarch.h

```diff
@@ -583,21 +583,21 @@ INST3(LAST_AVXVNNI_INSTRUCTION, "LAST_AVXVNNI_INSTRUCTION", IUM_WR, BAD_CODE, BA
 INST3(FIRST_BMI_INSTRUCTION, "FIRST_BMI_INSTRUCTION", IUM_WR, BAD_CODE, BAD_CODE, BAD_CODE, INS_TT_NONE, INS_FLAGS_None)
 
 // BMI1
-INST3(andn,             "andn",             IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0xF2),                             INS_TT_NONE,                                          REX_WX       | Encoding_VEX                   | INS_Flags_IsDstDstSrcAVXInstruction | Resets_OF    | Writes_SF    | Writes_ZF    | Undefined_AF | Undefined_PF | Resets_CF)    // Logical AND NOT
```

Comment:
We had specialized handling for these SSE38/SSE3A instructions since they have special prefixes; that handling wasn't really necessary and is better handled like this

Reply:
-- Looks like this bit is going to be slightly more involved, so I've pulled it out of the PR

Will see if I can get it resolved and up as its own thing later.